tox.ini

```diff
@@ -30,10 +30,7 @@ basepython =
     pypy3: pypy3
 
 [testenv:lint]
-basepython =
-    py35: python3.5
-    py36: python3.6
-    py37: python3.7
+basepython = python3.6
```

Comment:
Note that `[testenv:lint]` only does an exact match against the `lint` environment, so these rules were never applying, and no commands were being run in the py*-lint tests.